payment-processor/build.gradle

```diff
@@ -1,6 +1,20 @@
 plugins {
   id "org.jetbrains.kotlin.jvm" version "1.2.50"
   id "java-library"
+  id "idea"
```

Comment:
Do we need idea plugin ?

Reply:
Not needed, but nice to be able to run the test in IntelliJ